shared/constants/chat.js

```diff
@@ -266,6 +266,8 @@ export const StateRecord = Record({
   pendingFailures: Set(),
   conversationUnreadCounts: Map(),
   rekeyInfos: Map(),
+  alwaysShow: Set(),
```

Comment:
instead of 'youCreated' we instead just mark as 'alwaysShow' which is what we intend in the ui anyways